docs/en/compute/Hive.md

```diff
@@ -68,6 +68,12 @@ $ ./bin/alluxio fs copyFromLocal /path/to/ml-100k/u.user alluxio://master_hostna
 View Alluxio WebUI at `http://master_hostname:port` and you can see the directory and file Hive
 creates:
 
+> Note: Since the test data files are in text, make sure Hive is configured to read from TextFile. 
```

Comment:
- "are in text **format**..."
- "to read from TextFile for **managed and unmanaged tables**"?